pkg/kubelet/cadvisor.go

```diff
@@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
 	if cc == nil {
 		return nil, nil
 	}
-	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
+	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
```

Comment:
Design question - do we expect that kubelet supports a single container runtime or that it supports rocket and docker at the same time on the same machine?  If the former, how will we spec which one to run?  If the latter, do we support them by having kl.dockerClient and kl.rocketRuntime as 2 fields or do we hide them behind a runtime mux?